arm:
  kinematics_solver: kdl_kinematics_plugin/KDLKinematicsPlugin
  kinematics_solver_search_resolution: 0.005
  kinematics_solver_timeout: 0.005
  kinematics_solver_attempts: 3
active_hand:
  kinematics_solver: hsrb_moveit_kinematics/HSRBKinematicsPlugin
  kinematics_solver_search_resolution: 0.005
  kinematics_solver_timeout: 0.1
  kinematics_solver_attempts: 10
  # arm_lift_link, arm_flex_joint, arm_roll_joint, wrist_flex_link, wrist_roll_link, odom_x, odom_y, odom_t
  joint_weight: [1000.0, 100.0, 1.0, 1.0, 1.0, 100.0, 10.0, 1.0]
whole_body:
  kinematics_solver: hsrb_moveit_kinematics/HSRBKinematicsPlugin
  kinematics_solver_search_resolution: 0.005
  kinematics_solver_timeout: 0.005
  kinematics_solver_attempts: 3
  joint_weight: [10.0, 1.0, 1.0, 1.0, 1.0, 10.0, 10.0, 100.0]
whole_body_weighted:
  kinematics_solver: hsrb_moveit_kinematics/HSRBKinematicsPlugin
  kinematics_solver_search_resolution: 0.005
  kinematics_solver_timeout: 0.005
  kinematics_solver_attempts: 3
  joint_weight: [10.0, 1.0, 1.0, 1.0, 1.0, 10.0, 10.0, 100.0]
whole_body_light:
  kinematics_solver: hsrb_moveit_kinematics/HSRBKinematicsPlugin
  kinematics_solver_search_resolution: 0.005
  kinematics_solver_timeout: 0.005
  kinematics_solver_attempts: 3
  joint_weight: [10.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 100.0]
